mesonbuild/msubprojects.py

```diff
@@ -441,6 +441,7 @@ def update(self) -> bool:
             success = self.update_svn()
         elif self.wrap.type is None:
             self.log('  -> Cannot update subproject with no wrap file')
+            success = True
```

Comment:
I think we should only do this if we are updating "all the wraps", but still consider it an error if a single wrap was passed on the command line and it cannot be updated.

Reply:
fair enough, changed